hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/prefetch/TestS3ARemoteInputStream.java

```diff
@@ -288,4 +290,11 @@ private void testCloseHelper(S3ARemoteInputStream inputStream, int bufferSize)
     // Verify a second close() does not throw.
     inputStream.close();
   }
+
+  private static void assertAvailable(int expected, InputStream inputStream)
+      throws IOException {
+    assertThat(inputStream.available())
+        .describedAs("Check available bytes")
```

Comment:
in description, include inputStream string value, as it should be reporting its iostats

Reply:
Unfortunately, `S3ARemoteInputStream.toString()` only reports things like `nextReadPos` and `fpos` (no iostats!). Still useful, so I've included it anyway. But happy to also review/improve `toString()` if required, here or on a separate PR. 

Note that:
* this PR already modifies `toString()`, but only to take the lazy seek logic into account (and make it consistent across `S3ARemoteInputStream` and its sub-classes)
* `S3AInputStream` also does not seem to report iostats (unless I'm missing something)

Reply:
`S3AInputStream.toString()`:

```
String s = streamStatistics.toString();
```

the cloudstore bandwidth command does the toString calls if you ask for a -verbose run, which lets it print stats while still compiling against older releases